Dockerfile

```diff
@@ -2,10 +2,13 @@ FROM centos:centos7
 
 EXPOSE 8080
 
+RUN curl --retry 999 --retry-max-time 0 -sSL https://github.com/openshift/origin/releases/download/v1.2.0/openshift-origin-client-tools-v1.2.0-2e62fab-linux-64bit.tar.gz | tar xzv && \
+  mv openshift-origin-*/* /usr/bin/
```

Comment:
> What's the best practise with multiple RUN instructions in a Dockerfile?

The best practice is to combine them as much as possible.
In general, we should look for having the least number of layers possible, so this practice extends to the use of other instructions as well, such as EXPOSE, ENV, ADD, ...